notes-webapp/src/main/webapp/vue-app/notes-editor/components/NotesEditorDashboard.vue

```diff
@@ -422,7 +422,6 @@ export default {
             // delete draft note
             const draftNote = JSON.parse(localStorage.getItem(`draftNoteId-${this.note.id}`));
             this.deleteDraftNote(draftNote, notePath);
-            window.location.href = notePath;
```

Comment:
So after saving the note and deleting the draft, no redirection to the created note?

Reply:
The redirection is already set inside the deleteDraftNote(draftNote, notePath) function call.